### PR TITLE
Fix continuous monitor bugs.

### PR DESCRIPTION
**Pull request by ccanel**
Fixes two bugs in `parse_continuous_monitor.py`:
1.  Unable to find `"Running Disk Monotasks"` in Spark JSON data.
2.  Missing import: `argparse`

### CODE REVIEW

File: plot_continuous_monitor.py [hunk 1]
```diff
@@ -77,16 +78,17 @@ def plot_continuous_monitor(filename, open_graphs=False):
     running_compute_monotasks = 0
     if "Running Compute Monotasks" in json_data:
       running_compute_monotasks = json_data["Running Compute Monotasks"]
+    xvdf_running_disk_monotasks = 0
+    xvdb_running_disk_monotasks = 0
+    if "Running Disk Monotasks" in json_data:
```
Comment (kayousterhout) [inline plot_continuous_monitor.py:83]: Why is this necessary? Are you trying to parse old logs?

Comment (ccanel) [inline plot_continuous_monitor.py:83]: I am trying to parse logs from Spark, but Spark's ContinuousMonitor does not log "Running Disk Monotasks", so I added the above check to make sure that plot_continuous_monitor.py does not throw a key error. The code that I added is of the same form as the code for extracting "Running Compute Monotasks", "Running Macrotasks", etc.

Comment (kayousterhout) [inline plot_continuous_monitor.py:83]: https://github.com/NetSys/spark-monotasks/blob/master/core/src/main/scala/org/apache/spark/performance_logging/ContinuousMonitor.scala#L86

?

Comment (ccanel) [inline plot_continuous_monitor.py:83]: That's for the branch master. I'm parsing logs created by the branch spark_with_logging:
https://github.com/NetSys/spark-monotasks/blob/spark_with_logging/core/src/main/scala/org/apache/spark/performance_logging/ContinuousMonitor.scala#L61

Comment (kayousterhout) [inline plot_continuous_monitor.py:83]: Ah ok cool that makes sense
